output_3.2.txt

```diff
@@ -0,0 +1,8 @@
+TASK: 3.2
+  db.restaurants.aggregate([{$project: {gradeMax: { $max: "$grades.score"}, "restaurant_id":"$restaurant_id"}},{$sort:{gradeMax:-1}},{$limit:1}])
+{ "_id" : ObjectId("5cda658bcc4dd701131cc1e3"), "gradeMax" : 86, "restaurant_id" : "50000040" }
+
```

Comment:
I do not think that aggregate is the best choice for that task

Reply:
I am not sure, I have tried several ways, I didn't get the conclusion, Please give some hint

Reply:
Please find solution without agregate. I gave you a hint in slack

Reply:
I have Done

Reply:
I have Updated with solution